package/MDAnalysis/lib/distances.py

```diff
@@ -1351,14 +1586,35 @@ def calc_angle(a, b, c, box=None):
 
 
 def calc_dihedral(a, b, c, d, box=None):
-    """Dihedral angle (in degrees) between planes (a, b, c) and (b, c, d)
+    """Calculates the dihedral angle (in degrees) between the planes spanned by
+    the coordinates (`a`, `b`, `c`) and (`b`, `c`, `d`)::
+
+                  d
+                  |
+            b-----c
+           /
+          a
+
+    If the optional argument `box` is supplied, periodic boundaries are taken
+    into account when constructing the connecting vectors between coordinates,
+    i.e., the minimum image convention is applied for the vectors forming the
+    dihedral angle. Either orthogonal or triclinic boxes are supported.
 
     Parameters
     ----------
-    a, b, c, d : numpy.ndarray
-        single coordinate vectors
+    a,b,c,d : numpy.ndarray
+        Single coordinate vectors of shape ``(3,)``.
     box : numpy.ndarray, optional
-        simulation box, if given periodic boundary conditions will be applied
+        The unitcell dimensions of the system, which can be orthogonal or
+        triclinic and must be provided in the same format as returned by
+        :attr:`MDAnalysis.coordinates.base.Timestep.dimensions`:\n
+        ``[lx, ly, lz, alpha, beta, gamma]``.
+
+    Returns
+    -------
+    dihedral : float
+        The dihedral angle between the planes spanned by the coordinates
+        (`a`, `b`, `c`) and (`b`, `c`, `d`).
```

Comment:
Maybe add a 
```reST
See Also
--------
calc_dihedrals : calculate multiple dihedrals efficiently
```

Reply:
Same as above. The only difference between `calc_dihedral()` and `calc_dihedrals()` is that the former returns degrees and the latter radians. We should probably deprecate `calc_dihedral()`.